library/src/morpheus-master/sass/modules/grader/_base.scss

```diff
@@ -89,16 +89,15 @@ sakai-grader {
         .document-link { margin-bottom: 8px; font-weight: bold; }
 
         .preview-outer {
-          background: gray;
+          background: var(--sakai-background-color-3);
           text-align: center;
           padding: 30px;
           .preview-middle {
-            background: black;
-            background: white;
+            background: var(--sakai-background-color-1);
           }
           .preview-inner {
             width: 75%;
-            background: white;
+            background: var(--sakai-background-color-1);
             margin-top: 20px;
             margin-bottom: 20px;
             display: inline-block;
```

Comment:
This is outside of the scope of this PR, but I just noticed this here that this padding is missing a unit of pixels for its values.